wg-advisors/README.md

```diff
@@ -102,4 +102,37 @@ the ASF. Trust is transitive - that is, you may not know all members
 personally, but each member was nominated and elected by people that
 you, in turn, nominated and elected.
 
+### How to initiate PMC <-> Advisor relationship?
 
+There is no formal process for this. As an advisor, you may simply
+start participating in the project's mailing lists (dev and private)
+after introducing yourself and asking if this is fine for you to become
+an Advisor. The PMC must be allowed to say "no thank you" without providing any
+reason, and you must respect that decision, and not offer again unless
+invited.
+
+You can also be invited by the PMC to become an Advisor. When you become
+an Advisor, you should sign yourself up to the list of advisors in this
+repository (TODO: how) and describe your experience, skills and what
+you can bring as an Advisor (TODO: how).
+
+The ASF Board can also - during regular Board project review process - to
+identify a project that could benefit from an Advisor, and ask the PMC
+to consider inviting an Advisor, and cc: advisors mailing list (TODO: add the list)
+to make the advisors aware of such a project (which might also initiate
+a voluntary Advisor to step up and offer their help). In general the PMC
+should be allowed to decide whether they want an Advisor, and who that
+Advisor could bem but if any Advisor is interested in a project, they
+are encouraged to reach out to the PMC and offer their help.
+
+In neither case, the Board is appointing an Advisor. The Board is only
+asking the PMC to consider inviting an Advisor, and cc: advisors mailing
+list to make the advisors aware of such a project.
+
+### How to stop being an Advisor?
+
+You may stop being an Advisor at any time. You should inform the PMC
+that you are stepping down, and then unsubscribe from the private@ list
+and devlist. You do not need to provide any reason for stepping down,
```

Comment:
Any member can be subscribed to any project's dev@ or private@ list without giving justification for doing so. That's actually in the bylaws. We cannot mandate that a member unsub from these lists.

Reply:
Good point. I think maybe we should work out a way to just have a list of people who are advisors for PMC? Maybe in this repository ?

Reply:
Addressed it. Left TODO: on how to add youself as an advisor, but left the unsubsribe out.

Reply:
> Good point. I think maybe we should work out a way to just have a list of people who are advisors for PMC? Maybe in this repository ?

Yeah, I like the idea of tracking this -- not least because it might discourage multiple Advisors "ganging up" on one project.

Reply:
> Yeah, I like the idea of tracking this -- not least because it might discourage multiple Advisors "ganging up" on one project.

Yep. We just need to figure out the most light-weight way of tracking it and keeping it up-to-date. It would be useful on many levels to keep track of who is an advisor to which project,

Reply:
Perhaps, in the spirit of Minimal Viable Solution, a simple alpha-sorted list of projects, and the ApacheID of their Advisor, right here in this repo.

Reply:
Added a proposal for such a table.